functions/src/api/auth/assertValidToken.spec.ts

```diff
@@ -0,0 +1,46 @@
+// tslint:disable-next-line
+import '../../testUtils/mockFirebaseAdminAndFunctions'
+import { firestoreGet } from '../../testUtils/firestoreStub'
+import { JWTData } from '../../types/JWT'
+import { assertValidToken } from './assertValidToken'
+import { ForbiddenError } from '../errors/HttpError'
+
+describe('assertValidToken', () => {
+    const token = 'tokenValue'
+    const jwtData: JWTData = {
+        uid: '69',
+        iat: 1544455,
+    }
+
+    it('should not throw exception because token is valid in DB', async () => {
+        const snapshot = {
+            data: () => ({
+                tokens: {
+                    [token]: {
+                        valid: true,
+                    },
+                },
+            }),
+            exists: true,
+        }
+        firestoreGet.mockImplementation(() => Promise.resolve(snapshot))
+
+        const result = await assertValidToken(jwtData, token)
+
+        expect(result).toBeUndefined()
```

Comment:
please consider putting a try/catch block here as below. The catch block could have a failing expect (so that if the program reaches the catch, the test will fail), whereas the try block could have a passing expect, proving no exception was raised

Reply:
the test will fail if the exception is raised anyway.

Reply:
That's nice. Although I still feel it is strange to test the return value for a function that does not return anything :)
What if we just return the promise (`return assertValidToken(...)`) like in [this post](https://stackoverflow.com/questions/54525147/expect-a-jest-test-to-resolve-but-dont-care-about-the-value) ?